### ISSUE TITLE
Determine the targeted OSes

### ISSUE THREAD

**Issue by hamid-amar** [dev-team-fcc/SystemUsageAnalyzer#4]
I think, we'll be targeting linux and windows.

Leave your thoughts in here.

**Comment by sajanki** [comment]
I agree with this. Maybe Mac OS if someone is using that but I don't use it so I can't help in case of problems.
